src/coreComponents/physicsSolvers/contact/SolidMechanicsEFEMKernelsBase.hpp

```diff
@@ -293,6 +293,8 @@ class EFEMKernelsBase :
     // transp(B)DB
     LvArray::tensorOps::Rij_eq_AikBjk< nUdof, 3, 6 >( Kuw_gauss, matBD, compMatrix );
 
+    /// FIX: add old Equilibrium operator times oldStress (in Voigt notation)
```

Comment:
` LvArray::tensorOps::Rij_eq_AikBjk< 3, 3, 6 >( rw, eqMatrix, oldStressVoigt );`

Reply:
and then change `uLocal` in this kernel to be the increment.